### ISSUE TITLE
dont download, its malicious

### ISSUE THREAD

**Issue by helmutrocco13** [An00nymushun/DiscordFreeEmojis#74]
malicious

**Comment by helmutrocco13** [comment]
for context: when i used it it wrote a message saying something like: "instead of using my problem solving skills, i downloaded a virus of the internet"

**Comment by Zzeyy** [comment]
Then you haven't downloaded it from here.

**Comment by Esterze** [comment]
It's only your fault and your issue makes this awesome plugin look like a malware, when it's actually not.
I've been using this for months, maybe years? No problems so far.

What I think **actually** happened is that you download the plugin from somewhere,
just because you had some problem with the plugin, but you downloaded a plugin, that contained a malware.

Thankfully all your files are safe, if you removed the plugin and changed your password. Back to this later.

Then you came to whine about it here, because you want someone to blame for, sorry, it's all on you.
It looks like the person behind the malware did not want to hurt you, probably just troll you a bit.

I believe that running a plugin containing a malware in Discord threatens only your account.
But what's possible is that the malware gets out to your computer - then you must reinstall your computer.
But I bet that it was just a troll. Remove it, solve your problem, whatever it is and make sure to download files from verified sources.

**Comment by Esterze** [comment]
TLDR; This person download the plugin from somewhere else and the downloaded file contained a malware.
This plugin is just fine to use, I use it myself for some time and I'll keep using it.

**Comment by helmutrocco13** [comment]
ye makes sense that i download it from somewhere else and then come to the github to complain 🙄

**Comment by An00nymushun** [comment]
You definitely didn't download that from here
